### PR TITLE
Rolling Stock: play test fixes and tweaks

### PR DESCRIPTION
**Pull request by roseundy**
- A few bug fixes
- Many UI tweaks
- Add Master-Mode support and improve hotseat mode

### CODE REVIEW

File: assets/app/view/game_page.rb [hunk 1]
```diff
@@ -443,7 +448,11 @@ def render_game
       children << h(Game::EntityOrder, round: @round)
       unless @game.finished
         children << h(Game::Abilities, user: @user, game: @game)
-        children << h(Game::Pass, actions: current_entity_actions)
+        children << if @game.round.unordered? && hotseat_or_master
+                      h(Game::MasterPass)
```
Comment (tobymao) [inline assets/app/view/game_page.rb:452]: why is this needed?

Comment (roseundy) [reply assets/app/view/game_page.rb:452]: In rounds where multiple players may act at any point ("unordered"), while in hotseat or master_mode, I need to display multiple "pass" buttons - one per player that can still act.

Comment (roseundy) [reply assets/app/view/game_page.rb:452]: RS_master

Comment (roseundy) [reply assets/app/view/game_page.rb:452]: Outside of master_mode:

RS_nomaster
